bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/common.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="signal">
+		<label>Signal</label>
+		<channels>
+			<channel id="value" typeId="rssi"/>
+			<channel id="strength" typeId="system.signal-strength"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="battery">
+		<label>Battery</label>
+		<channels>
+			<channel id="value" typeId="system.battery-level"/>
+			<channel id="low-battery" typeId="system.low-battery"/>
```

Comment:
It seems that descriptions are missing for these and some other channels with system types (noticed also for signal#strength).

Reply:
Agree, but I think it would be better to address in DefaultSystemChannelTypeProvider instead that in each binding. Don't you think so ?

Reply:
Sure, if generic descriptions makes sense, this could eliminate some duplication. I see that you created openhab/openhab-core#2912. You can consider decoupling that PR into two: One for the description texts and one for the null checks - to speed up the review process. :)